### DISCUSSION TITLE
Auto Run Streaming

### DISCUSSION THREAD

**Discussion by tips2fix** [sorayuki/obs-multi-rtmp#70] [Q&A]
First of all great job with this plugin, 

is that possible for this plugin to start streaming automaticly when obs open up?

im able to do this process with OBS  .but file : with this code
start /d "C:\Program Files\obs-studio\bin\64bit" obs64.exe --startstreaming

and this start streaming for obs studio but how i can do this to for your plugin?

**Comment by pullasurya** [comment]
Hi guys,
I installed RTMP Plugin. but I would like to make it Automatic run and stop. kindly welcome your suggestions.
thanks
